requirements.txt

```diff
@@ -6,4 +6,5 @@ urllib3==1.26.5
 pika==1.2.0
 Flask==2.0.1
 flask-cors==3.0.10
-PyJWT==2.3.0
\ No newline at end of file
+PyJWT==2.3.0
+lark==0.11.3
```

Comment:
Just in case someone asks, lark has MIT license: https://www.npmjs.com/package/lark